app/src/assets/css/modules/_section.scss

```diff
@@ -209,7 +209,10 @@
         }
 
         & .img {
-            max-height: 33rem;
+            max-height: 40rem;
+            @include respond(phone) {
+                max-height: none;
+            }
```

Comment:
Did we check if these changes did not affect existing UI's?

Reply:
Currently `ImageDetailView.vue` is the only page that uses this specific scss class (`section_md-header` > `.img`). I can change this though if it's going to be a concern